From 5eb321f779b84f74a85834b28561d3695a0dc2ea Mon Sep 17 00:00:00 2001
From: Ashok Kumar <ashok2.kumar@amd.com>
Date: Fri, 29 Jul 2022 09:07:22 -0500
Subject: [PATCH] Linux-aspeed: Fix for PVDD11_S3_P1 VR Titanite sensor

Typo mistmatch caused the issue to not populate PVDD11_S3_P1 VR sensor
Fixes: Fix for PVDD11_S3_P1 VR Titanite sensor

Signed-off-by: Ashok Kumar <ashok2.kumar@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
index 9b7bad9d01a5..c9ce6bf393bb 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-titanite.dts
@@ -635,7 +635,7 @@ pvddio_p1@75 {
            compatible = "isil,isl68137";
            reg = <0x75>;
        };
-       pvddcr_s3_p1@74 {
+       pvdd11_s3_p1@74 {
            compatible = "isil,isl68137";
            reg = <0x74>;
        };
--
2.25.1

